tensorboard/plugins/distributions/tf_distribution_dashboard/tf-distribution-dashboard.html

```diff
@@ -71,61 +70,117 @@
       </div>
 
       <div class="center">
-        <tf-panes-helper
-          categories="[[_categories]]"
-          data-type="[[dataType]]"
-          data-provider="[[dataProvider]]"
-          data-not-found="[[dataNotFound]]"
-          run2tag="[[run2tag]]"
-          selected-runs="[[_selectedRuns]]"
-          repeat-for-runs
+        <template is="dom-if" if="[[_dataNotFound]]">
```

Comment:
We had this: `tf-no-data-warning`. We removed it because it made too many assumptions about the structure of the data. There are conventions that the first-party plugins follow, but they're not requirements, so we didn't want to expose an API that depended on them.

I'm open to rethinking this if you want.